src/style.css

```diff
@@ -1 +1,12 @@
 /* styles go here */
+.form {
+  margin-top: 10px;
+}
+
+.form-field {
+  margin-top: 20px;
+}
+
+.password {
+  margin-top: 10px;
```

Comment:
there are several classes with same property  `margin-top: 10px;`, combine it together and use only one class